### PR TITLE
item_groups sorting

### PR DESCRIPTION
**Pull request by LilShiningMan**
```SUMMARY: Content "item spawn list sorting"```

#### Summary
sorted more lists from item_groups.json into proper categorical .jsons

#### Purpose of change
Tidying

#### Describe the solution
Took various lists and sorted in to locations, locations_commercial and vending_machines.json lists.

#### Testing
De-linted and debug spawned in-game.

### CODE REVIEW

**Comment by LilShiningMan** [comment]
Yeah, I had the same thought but figured I'd keep it simple for now. I do like the idea of having things broken down in more distinct categorical .jsons but wasn't sure of devs' thoughts on having a lot of changes in same pr (ie- moving mass amounts of lists into new files).  If that's what's wanted I can def do that going forward.

**Comment by anothersimulacrum** [comment]
You have merge conflicts

**Comment by LilShiningMan** [comment]
Thanks for the heads up, will fix tonight.

**Comment by LilShiningMan** [comment]
When I checked the conflict, it marked 4 lists in item_groups in yellow that shouldn't have been there as they were moved as part of this PR to locations.json. I removed these and hit "resolve conflict". Now the travis build has failed. I'm not exactly savvy when it comes to github, so I'm a little lost as to what I should be doing to resolve this. Any advice?

**Comment by anothersimulacrum** [comment]
The itemgroups you have here have items that were removed recently (`bobburger`, `cheeseburgerhuman`). Remove those.

File: data/json/itemgroups/locations.json [hunk 1]
```diff
@@ -2324,5 +2324,359 @@
...
+      [ "RPG-7_tbg7v", 1 ],
+      [ "RPG-7_pg7vr", 1 ],
+      [ "clown_suit", 2 ],
+      [ "clownshoes", 2 ],
+      [ "bondage_suit", 1 ],
+      [ "gold_dental_grill", 1 ],
+      [ "knife_trench", 14 ],
+      [ "switchblade", 15 ],
+      [ "bondage_mask", 1 ],
+      [ "pie_human", 5 ],
```
Comment (ZhilkinSerg) [inline data/json/itemgroups/locations.json]:
```suggestion
```

**Comment by LilShiningMan** [comment]
thanks for all the help. If anything else needs resolving I can look again tomorrow.

**Comment by LilShiningMan** [comment]
Passed checks and looks OK to me. Let me know if I need to do anything else.

**Pull request merged**